recipes/sbpipe/meta.yaml

```diff
@@ -8,11 +8,11 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
-  sha256: 5ae5fe0332ea76567f9f5405abda27f2631146ff6c735ac809127939f61c9fac
+  sha256: c417db041e8990f41bc56101b7a4be7a861136bac67207dec47b83546ca26d50
 
 build:
   noarch: python
-  number: 1
+  number: 2
```

Comment:
Why is the build number not being reset to 0? After all this is a new version, right?

Reply:
Yes, it is a new version of the same package. Sorry, for some reason I thought the number had to be incremental every time there is a PR for the same package.